lib/ruby_lsp/requests/support/rubocop_runner.rb

```diff
@@ -3,6 +3,7 @@
 
 begin
   require "rubocop"
+  return if Gem.loaded_specs["rubocop"].version < Gem::Version.create("1.4.0")
```

Comment:
Can we instead use `Kernel.gem` here to do this?

```suggestion
  return unless gem("rubocop", ">= 1.4.0")
  require "rubocop"
```

Reply:
When I tried this, it errored:
```
can't activate rubocop (>= 1.4.0), already activated rubocop-1.3.0. Make sure all dependencies are added to Gemfile.
```
I think this is because `gem(gem_name, *requirements)` is used to activate a specific version of `gem_name` and not to check for it? [source code](https://github.com/rubygems/rubygems/blob/cbcbcbbef98514b96c65309bc933363afd5879a6/lib/rubygems/core_ext/kernel_gem.rb)
I looked around on RubyGems for a cleaner solution, but couldn't find anything aside from solutions that were similar (e.g., using `Gem::Requirement` with the rubocop version through `Gem.loaded_specs` and `satisfied_by?`).

Reply:
Oh I see! Thanks for the clarification 😄 I updated the PR